tests/dummy/app/templates/docs/guides/development-and-debugging.md

```diff
@@ -0,0 +1,68 @@
+# Development and Debugging
+
+## require() and requireNode()
+
+TL;DR is to use `requireNode()` instead of `require()` when requiring node modules from your Ember application.
+
+Ember defines its own global `require` function for requiring transpiled Ember modules from the application bundle, which overwrites Electron's `require` function that is used to require Node.js modules out of `node_modules` or by relative path.
+
+`ember-electron` attempts to patch this up by implementing a `require()` function that tries both Ember's and Node.js's `require()` functions, but it's not perfect, so we recommend you use `requireNode()` when requiring Node.js modules.
+
+## Command-line arguments
+
+If your Electron app accepts command-line arguments that the main process handles via `process.argv`, then when running `ember electron` you need to precede them with `---`. This allows the `ember electron` to distinguish between its own arguments and arguments it should pass through when launching Electron:
+
+```sh
+$ ember electron <ember electron args> --- <args to pass to Electron>
+```
+
+e.g.
+
+```sh
+$ ember electron --- --some-flags --that-my-app --handles-as="process.argv"
+```
+
+or
+
+```sh
+$ ember electron --environment=production --- --some-flags --that-my-app --handles-as="process.argv"
+```
+
+## Ember Inspector and Devtron
+
+`ember electron` installs the Electron Inspector and Devtron directly into Electron, so they should be available if you open the dev tools. Note that if you're in a browser window without `nodeIntegration` enabled, they will not be installed.
+
+## Developing main process code
+
+`ember-electron` does not restart your Electron app when main process code changes, so after making a code change you need to quit and re-launch your application. If you just run `ember electron` each time and you aren't changing any Ember code, you'll have to unnecessarily wait through the Ember build process that `ember electron` automatically runs. To avoid this, you can
```

Comment:
```suggestion
`ember-electron` does not restart your Electron app when main process code changes, so after making a code change you need to quit and re-launch your application. If you only run `ember electron` each time and you aren't changing any Ember code, you'll have to unnecessarily wait through the Ember build process that `ember electron` automatically runs. To avoid this, you can
```

Reply:
Maybe it's stylistic, but this actually reads less clear to me. What's your argument against `just` in this context?

Reply:
I feel like `only === just` in meaning here, as the thing you are trying to convey is you only run `ember electron`.